modules/delegates.js

```diff
@@ -761,8 +761,11 @@ shared.getForgedByAccount = function (req, cb) {
 			return setImmediate(cb, err[0].message);
 		}
 
-		if (req.body.start && req.body.end) {
+		if (req.body.start !== undefined || req.body.end !== undefined) {
```

Comment:
We should allow `start/end` to be `0` and search by only one border, previous cond not allow that.